Doc/c-api/gcsupport.rst

```diff
@@ -173,3 +173,44 @@ if the object is immutable.
    this method (don't just call :c:func:`Py_DECREF` on a reference).  The
    collector will call this method if it detects that this object is involved
    in a reference cycle.
+
+
+Controlling the Garbage Collector State
+---------------------------------------
+
+The C-API provides the following functions for controlling
+garbage collection runs.
+
+.. c:function:: Py_ssize_t PyGC_Collect(void)
+
+   Perform a full garbage collection, if the garbage collector is enabled.
+   Returns the number of collected + unreachable objects which cannot
+   be collected.
+   If the garbage collector is disabled or already collecting,
+   returns ``0`` immediately.
+   Errors during garbage collection are passed to :data:`sys.unraisablehook`.
+   This function does not raise exceptions.
+
+
+.. c:function:: int PyGC_Enable(void)
+
+   Enable the garbage collector: similar to :func:`gc.enable`.
+   Returns the previous state, 0 for disabled and 1 for enabled.
+
+.. versionchanged:: 3.10
```

Comment:
Oh, the markup should be at the same indentation level than the paragraph. Also "versionadded" is better than "versionchanged".

```suggestion
   Returns the previous state, 0 for disabled and 1 for enabled.

   .. versionadded:: 3.10
```

Same remark for the 2 other added functions.

Reply:
Right. I had thought of it and then still ended up copynpasting… :-/